packages/connectors/src/blocto.ts

```diff
@@ -0,0 +1,165 @@
+   * To get advanced features and support with Blocto.
+   *
+   * https://docs.blocto.app/blocto-sdk/register-app-id
+   */
+  appId?: string
+}
+
+blocto.type = 'blocto' as const
+export function blocto({ appId }: BloctoParameters = {}) {
+  // eslint-disable-next-line @typescript-eslint/ban-types
+  type Properties = {}
+  type StorageItem = {
+    store: any
+    'wagmi.recentConnectorId': string
+  }
+
+  let walletProvider: BloctoProvider | undefined
+  const handleConnectReset = () => {
+    walletProvider = undefined
+  }
+
+  return createConnector<BloctoProvider, Properties, StorageItem>((config) => ({
+    id: 'blocto',
+    name: 'Blocto',
+    type: blocto.type,
+    async connect({ chainId } = {}) {
+      try {
+        const provider = await this.getProvider({ chainId })
+
+        config.emitter.emit('message', { type: 'connecting' })
+
+        await provider.request({
+          method: 'eth_requestAccounts',
+        })
+
+        const accounts = await this.getAccounts()
+        const _chainId = await this.getChainId()
+
+        return { accounts, chainId: _chainId }
+      } catch (error: unknown) {
+        handleConnectReset()
+        throw error
+      }
+    },
+    async disconnect() {
+      const provider = await this.getProvider()
+      await provider.request({ method: 'wallet_disconnect' })
+      handleConnectReset()
+    },
+    async getAccounts() {
+      const provider = await this.getProvider()
+      const accounts = (await provider.request({
+        method: 'eth_accounts',
+      })) as string[]
+
+      return accounts.map((x) => getAddress(x))
+    },
+    async getChainId() {
+      const provider = await this.getProvider()
+      const chainId = await provider?.request({ method: 'eth_chainId' })
+      return normalizeChainId(chainId)
+    },
+    async getProvider({ chainId } = {}) {
+      if (!walletProvider) {
+        const store = await config.storage?.getItem('store')
+        const lastConnectedChainId = store?.state?.chainId
+        const desiredChainId = chainId ?? lastConnectedChainId
+        const ethereum: BloctoEthereumProviderParameters = {
+          chainId: desiredChainId,
+          rpc: config.chains.find((x) => x.id === desiredChainId)?.rpcUrls
+            .default.http[0],
+        }
+
+        walletProvider = new BloctoSDK({ ethereum, appId })?.ethereum
+        if (!walletProvider) {
+          throw new Error('Blocto SDK is not initialized.')
+        }
+
+        walletProvider.on('accountsChanged', this.onAccountsChanged.bind(this))
+        walletProvider.on('chainChanged', this.onChainChanged.bind(this))
+        walletProvider.on('disconnect', this.onDisconnect.bind(this))
+      }
+
+      return Promise.resolve(walletProvider)
+    },
+    async isAuthorized() {
+      const recentConnectorId = await config.storage?.getItem(
+        'recentConnectorId',
+      )
+      if (recentConnectorId !== this.id) return false
```

Comment:
Curious why this is needed?

Reply:
Just checked, and it's no longer needed. The update of `@blocto/sdk` has solved the problem that originally needed to be checked.

It's been resolved in [fix(connector): optimize isAuthorized check condition in blocto](https://github.com/wevm/wagmi/pull/3625/commits/98f767c42afc42b336be7e2d2694a96281486147)